### PR TITLE
[SDK-3680] README redesign

### PR DESCRIPTION
**Pull request by jimmyjames**
Updates the README per the new design specs.

Notes:
- There was no existing `EXAMPLES.md`, so I created one and populated with the example content that was contained in the README
- The README had a lot of API-level documentation within it. I removed most of it, as it is covered in the API reference documentation now more clearly linked-to in the README.
- This PR does not the banner, as we are awaiting final design. We will either update this PR when it is available, or follow-up with an additional PR.

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+[Organizations](https://auth0.com/docs/organizations) is a set of features that provide better support for developers who build and maintain SaaS and Business-to-Business (B2B) applications.
+
+Using Organizations, you can:
+
+- Represent teams, business customers, partner companies, or any logical grouping of users that should have different ways of accessing your applications, as organizations.
+- Manage their membership in a variety of ways, including user invitation.
+- Configure branded, federated login flows for each organization.
+- Implement role-based access control, such that users can have different roles when authenticating in the context of different organizations.
+- Build administration capabilities into your products, using Organizations APIs, so that those businesses can manage their own organizations.
+
```
Comment (frederikprijck) [inline EXAMPLES.md:103]:
```suggestion

```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: EXAMPLES.md (hunk 1 above)
Comment (frederikprijck) [reply EXAMPLES.md:103]: As mentioned on slack, we are getting rid of some of the organizations marketing text, as we are already point to the docs above.

Comment (frederikprijck) [reply EXAMPLES.md:103]: It looks like this is still present, is that intentional?

**Review by frederikprijck — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -334,443 +75,43 @@ You can use the Authentication API to obtain a token for a previously authorized
...
 
-## Author
+### Contributing
 
-[Auth0](https://auth0.com)
+We appreciate feedback and contribution to this repo! Before you get started, please see the following:
 
-## License
+- [Auth0's general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
+- [Auth0's code of conduct guidelines]((https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md))
```
Comment (ewanharris) [inline README.md:98]: Extra brackets here
```suggestion
- [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
```

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**